src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Shape {
+    private double top;
+    private double bottom;
+    private double height;
+
+    public IsoscelesTrapezoid(String color, double top, double bottom, double height) {
+        super(Colors.valueOf(color));
+        this.top = Math.round(Math.abs(top));
+        this.bottom = Math.round(Math.abs(bottom));
+        this.height = Math.round(Math.abs(height));
```

Comment:
Rounding the dimensions using `Math.round()` might not be necessary unless specified by the task requirements. Ensure this aligns with the intended functionality.